src/presentation/http/router/noteSettings.test.ts

```diff
@@ -305,19 +459,48 @@ describe('NoteSettings API', () => {
     });
 
     test('Generate invitation hash by public id with 200 status, user is creator of the note', async () => {
-      const userId = 2;
-      const accessToken = global.auth(userId);
+      /**
+       * truncate all tables, which are needed
+       * restart autoincrement sequences for data to start with id 1
+       *
+       * TODO get rid of restarting database data in tests (move to beforeEach)
+       */
+      await global.db.truncateTables();
+
+      /** create test user */
+      const user = await global.db.insertUser({
+        email: 'a@a.com',
+        name: 'Test user 1',
+      });
 
-      const userNote = notes.find(newNote => {
-        return newNote.creator_id === userId;
+      /** create test note for created user */
+      const note = await global.db.insertNote({
+        creatorId: user.userId,
+        publicId: 'Pq1T9vc23Q',
       });
 
+      /** create test note settings for created note */
+      await global.db.insertNoteSetting({
+        noteId: note.noteId,
+        isPublic: true,
+        invitationHash: 'Hzh2hy4igf',
+      });
+
+      /** create test team member for created note */
+      await global.db.insertNoteTeam({
+        userId: user.userId,
+        noteId: note.noteId,
+        role: 0,
+      });
+
+      const accessToken = global.auth(user.userId);
+
       const response = await global.api?.fakeRequest({
         method: 'PATCH',
         headers: {
           authorization: `Bearer ${accessToken}`,
         },
-        url: `/note-settings/${userNote!.public_id}/invitation-hash`,
+        url: `/note-settings/Pq1T9vc23Q/invitation-hash`,
       });
 
       expect(response?.statusCode).toBe(200);
```

Comment:
there also should be an assertion that the new hash is not equals the previous one